qiskit/pulse/reschedule.py

```diff
@@ -177,7 +177,7 @@ def add_implicit_acquires(schedule: ScheduleComponent, meas_map: List[List[int]]
 def pad(schedule: Schedule,
         channels: Optional[Iterable[Channel]] = None,
         until: Optional[int] = None) -> Schedule:
-    """Pad the input ``Schedule`` with ``Delay``s on all unoccupied timeslots until ``until``
+    """Pad the input ``Schedule`` with ``Delay``\s on all unoccupied timeslots until ``until``
```

Comment:
```suggestion
    """Pad the input ``Schedule`` with ``Delay`` s on all unoccupied timeslots until ``until``
```
Looks like we have to go back to my original PR